crates/biome_cli/tests/commands/lint.rs

```diff
@@ -990,11 +990,11 @@ fn fs_error_unknown() {
 // │   └── test.js  // ok
 // └── src
 //     ├── symlink_testcase1_1 -> hidden_nested
+//     ├── symlink_testcase1_3 -> hidden_testcase1/test/test.js
```

Comment:
It seems you forgot add this symbol link to the test

Reply:
It is already in place. The comment was outdated.